.github/workflows/cd-prod-ios.yaml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to Expo
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+    branches:
+      - dev
+jobs:
+  expo-build:
+    runs-on: ubuntu-latest
+    environment: development
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: yarn
+
+      - name: 🏗 Setup EAS
+        uses: expo/expo-github-action@v8
+        with:
+          eas-version: latest
+          token: ${{ secrets.EXPO_TOKEN }}
```

Comment:
TODO: (to be taken once `dev` and `main` PRs are merged in)
- update Readme.md, somewhere in CD section, we should mention the significance of expo token and it should be present in the secrets in order the cd workflows to succesfuly run